src/set_theory/game/pgame.lean

```diff
@@ -899,34 +899,19 @@ def relabelling.neg_congr : ∀ {x y : pgame}, x.relabelling y → (-x).relabell
     λ i, relabelling.neg_congr (by simpa using R_relabelling (R_equiv i)),
     λ i, relabelling.neg_congr (by simpa using L_relabelling (L_equiv.symm i))⟩
 
-@[simp] theorem neg_le_neg_iff : Π {x y : pgame}, -y ≤ -x ↔ x ≤ y
+private theorem neg_le_lf_neg_iff :
```

Comment:
I think this would be a better name
```suggestion
private theorem neg_le_iff_and_neg_lf_iff :
```

Reply:
With the conventions below it would be `neg_le_neg_iff_and_neg_lf_neg_iff`, which feels too long; I figured since it's private the name doesn't matter and opted for the current name `neg_le_lf_neg_iff`.